roles/deprecated_api/tasks/get_api_request_counts_per_namespace.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "Get API request counts for a workload ns {{ deprecated_ns }}"
```

Comment:
Nice findings about the APIRequestCount large object size!

In regards to that, in the successful job you reference it, I see the tasks `Get API request counts for a workload ns *` print a large output[1] and that is making the ansible.log heavier, would it be possible to add `no_log: true` here? Thanks

[1]
https://www.distributed-ci.io/jobs/21b70140-5c74-4bdb-ac9b-07be43d2278d/jobStates?sort=date&task=a490a05a-397c-4003-8e8c-a27ce35b2039
https://www.distributed-ci.io/jobs/21b70140-5c74-4bdb-ac9b-07be43d2278d/jobStates?sort=date&task=05b244ed-6628-4ca5-8ea1-bc97fbb66169

Reply:
Hi @manurodriguez, thank you for the review and the nice idea! It looks logical, I don't know why I did not think of it. I pushed the change.